src/perform_step/firk_perform_step.jl

```diff
@@ -1,4 +1,4 @@
-function do_newJ(integrator, alg, cache, repeat_step)::Bool # for FIRK
+  function do_newJ(integrator, alg, cache, repeat_step)::Bool # for FIRK
```

Comment:
mistake?

Reply:
For some reason VScode does not highlight diffs in the first line. So this one went trough.